.github/workflows/sims.yml

```diff
@@ -84,7 +84,7 @@ jobs:
     needs:
       [test-sim-multi-seed-short, test-sim-after-import, test-sim-import-export]
     runs-on: ubuntu-latest
-    if: ${{ success() }}
+    if: ${{ false }} # Disabled due to requiring Slack integration
```

Comment:
It seems ideal to continue to run the sim tests above, but these two jobs require a `SLACK_WEBHOOK` which pipes failures and successes into slack. I think this would just be noise for us for now so I'd prefer not to do this.